spec/paper_trail/model_spec.rb

```diff
@@ -877,4 +877,55 @@
       expect(widget.versions.empty?).to(eq(true))
     end
   end
+
+  context "without the object column" do
+    # rubocop:disable RSpec/BeforeAfterAll
+    before :all do
+      ActiveRecord::Migration.remove_column :versions, :object
+      PaperTrail::Version.reset_column_information
+    end
+    after :all do
+      ActiveRecord::Migration.add_column :versions, :object, :text
+      PaperTrail::Version.reset_column_information
+    end
+    # rubocop:enable RSpec/BeforeAfterAll
```

Comment:
Normally I'd avoid doing this sort of thing in a `before(:each)` b/c that'd mean doing four extra database requests per test (removing the column, adding the column, and 2x reloading the schema).

It appears that the Rubocop error is concerned with transactional fixtures in Rails:
```
spec/paper_trail/model_spec.rb:882:5: C: RSpec/BeforeAfterAll: Beware of using before :all as it may cause state to leak between tests. If you are using rspec-rails, and use_transactional_fixtures is enabled, then records created in before :all are not automatically rolled back.
    before :all do
    ^^^^^^^^^^^
spec/paper_trail/model_spec.rb:886:5: C: RSpec/BeforeAfterAll: Beware of using after :all as it may cause state to leak between tests. If you are using rspec-rails, and use_transactional_fixtures is enabled, then records created in after :all are not automatically rolled back.
    after :all do
```
So perhaps we should disable this cop globally.

Reply:
That research ended up being fairly inconclusive. I'll update this to use `before(:each)` if you'd still prefer that approach.